CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ conda create --name pyani-plus_py312 python=3.12 -y
 conda activate pyani-plus_py312
 conda config --add channels bioconda
 conda config --add channels conda-forge
-conda config --set channel_priority strict
+conda config --set channel_priority flexible
```

Comment:
I don't receall using this setting myself - not sure what the implications are?

Reply:
Ah - cross reference https://github.com/pyani-plus/pyani-plus/pull/347#issuecomment-2844652969